### ISSUE TITLE
datepicker-ionic2@2.6.0 not worked on Android with --prod mode

### ISSUE THREAD

**Issue by EvPut** [misha130/datepicker-ionic2#41]
datepicker-ionic2@2.6.0 does not work on Android with --prod mode
Build success, but when in app datepicker is called, in console:
Error: Uncaught (in promise): Error: No component factory found for undefined. Did you add it to @NgModule.entryComponents?

Without "--prod" works fine.

Thanks in advance for the help.

```
cli packages:
    @ionic/cli-utils  : 1.9.2
    ionic (Ionic CLI) : 3.9.2

global packages:
    Cordova CLI : 7.0.1

local packages:
    @ionic/app-scripts : 2.1.4
    Cordova Platforms  : android 6.2.3 ios 4.4.0
    Ionic Framework    : ionic-angular 3.6.0

System:
    Android SDK Tools : 25.2.5
    Node              : v7.10.0
    npm               : 4.5.0
    OS                : Windows 10
```

**Comment by thedang** [comment]
Me too, not only for android, when build with command :
npm run build --prod
throw error Error: Uncaught (in promise): Error: No component factory found for undefined. Did you add it to @NgModule.entryComponents?

**Comment by miheyinfo** [comment]
+1

**Comment by EvPut** [comment]
I tried different options, removed lazy loading pages, it still does not work with --prod.

**Comment by misha130** [comment]
I should have a new version, today or tomorrow. Its a problem with the
ionic modals that I imitate to display the picker.

**Comment by misha130** [comment]
Fixed in 2.6.1
